optuna/importance/__init__.py

```diff
@@ -87,4 +89,29 @@ def get_param_importances(
     if not isinstance(evaluator, BaseImportanceEvaluator):
         raise TypeError("Evaluator must be a subclass of BaseImportanceEvaluator.")
 
-    return evaluator.evaluate(study, params=params, target=target)
+    if target is None:
+        if study._is_multi_objective():
+            raise ValueError(
+                "If the `study` is being used for multi-objective optimization, "
+                "please specify the `target`. For example, use "
+                "`target=lambda t: t.values[0]` for the first objective value."
+            )
+
+        def default_target(trial: FrozenTrial) -> float:
+            val: Optional[float] = trial.value
+            assert val is not None
+            return val
+
+        target = default_target
+
+    if params is None:
+        params = list(intersection_search_space(study, ordered_dict=True).keys())
+
+    importances = evaluator.evaluate(study, params=params, target=target)
+
+    # Sort the importances in descending order.
+    return OrderedDict(
+        reversed(
+            sorted(importances.items(), key=lambda name_and_importance: name_and_importance[1])
+        )
+    )
```

Comment:
It would be informative to address that the returned dictionary is sorted by its values descendingly in the `Retuned` part of the docstring based on this change.